src/d3a/setup/kpi/d3asim_2103_2_sided_pab.py

```diff
@@ -0,0 +1,95 @@
+import os
```

Comment:
I was thinking about that too, unfortunately this is not possible for 2 reasons:
1. Most important reason, the one-sided market setup file is not exactly the same as the 2 sided market one. In order to reproduce consistently the same trade pattern in both markets, I had to alter the pricing of the strategies too. Therefore there have to indeed be 2 different files. 
2. These tests are using the step that executes the simulation via the console. Therefore there is no easy way for altering the market type (of course, we can set env variables, that we then read in the setup file, but this approach seemed clunky to me)

Reply:
Considering point 1, I would actually prefer to showcase the relatively change in trade thats is due to change in market that also lead to different KPI result. So others could could have a feeling of its affect due to change of market type with the same scenario.
Considering point 2, it could circumvented if we use this step `integration_tests/steps/integration_tests.py:611 --> we run the simulation with setup file {scenario} and parameters .. `

Reply:
Considering point 1, I disagree, and I think there is a misunderstanding in what the test does on your side. The purpose of the test is not to check for different KPI results according to the market type. The purpose of the test is to check that, assuming that the 2 markets produce the same trades, the self sufficiency and self consumption members do not report different values.

Reply:
Thanks for explanation agreed now with point 1. Regarding point 2 its optional as I don't see any difference